rust/01-hello-world/README.md

```diff
@@ -88,12 +97,12 @@ Next you need to point at or copy the libraries over:
 From the toolchain I installed it was installed here:
 
 ```
-cp /usr/local/Cellar/x86_64-unknown-linux-gnu/7.2.0/toolchain/x86_64-unknown-linux-gnu/sysroot/lib64/ld-linux-x86-64.so.2 .
-cp /usr/local/Cellar/x86_64-unknown-linux-gnu/7.2.0/toolchain/x86_64-unknown-linux-gnu/sysroot/lib64/libc.so.6 .
-cp /usr/local/Cellar/x86_64-unknown-linux-gnu/7.2.0/toolchain/x86_64-unknown-linux-gnu/sysroot/lib64/libdl.so.2 .
-cp /usr/local/Cellar/x86_64-unknown-linux-gnu/7.2.0/toolchain/x86_64-unknown-linux-gnu/sysroot/lib64/libgcc_s.so.1 .
-cp /usr/local/Cellar/x86_64-unknown-linux-gnu/7.2.0/toolchain/x86_64-unknown-linux-gnu/sysroot/lib64/libpthread.so.0 .
-cp /usr/local/Cellar/x86_64-unknown-linux-gnu/7.2.0/toolchain/x86_64-unknown-linux-gnu/sysroot/lib64/librt.so.1 .
+cp /usr/local/Cellar/x86_64-unknown-linux-gnu/7.2.0/toolchain/x86_64-unknown-linux-gnu/sysroot/lib64/ld-linux-x86-64.so.2 lib64
```

Comment:
maybe for explicit's sake could be lib64/.

Reply:
Good idea, force-pushed that into new `HEAD` @ 54cb64d